CatchMe/CatchMe/Source/Components/Main/mainReportView.swift

```diff
@@ -121,19 +121,19 @@ class mainReportView: UIView {
         
         titleLabel.text = "활동 리포트"
         titleLabel.textColor = .white
-        titleLabel.font = UIFont.systemFont(ofSize: 13, weight: .medium)
+        titleLabel.font = UIFont.stringMediumSystemFont(ofSize: 13)
```

Comment:
손배야 사실 안 바꿔도 되는데 고냥.. 요롷게 쓰면 앞으로 손이 더 편할 거 같다도라..
```suggestion
        titleLabel.font = .stringMediumSystemFont(ofSize: 13)
```

Reply:
....선배.... 그냥 이케 해도 되는거였구나.... 나 몰랐자너.............. 감삼다...💋